Wikipedia/Code/NotificationsCenterViewController.swift

```diff
@@ -184,6 +200,14 @@ extension NotificationsCenterViewController: UICollectionViewDelegate {
             viewModel.fetchNextPage()
         }
     }
+    
+    func collectionView(_ collectionView: UICollectionView, shouldBeginMultipleSelectionInteractionAt indexPath: IndexPath) -> Bool {
```

Comment:
@staykids just FYI this is probably not a must if it will interfere with your swipe action handling. I just added it to see if we can get any bonus functionality by using the system editing mode.